README.md

```diff
@@ -1,2 +1,243 @@
-# blockstack-ng
-Reference implementation of the Stacks blockchain v2
+# Stacks 2.0
+
+Reference implementation of https://blockstack.org/whitepaper.pdf in Rust.
```

Comment:
"is in Rust."

Reply:
I think it works either way -- the header of repo's readme is usually a "noun" description of what the repo is.

"This is the reference implementation of whitepaper in Rust."

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1251/commits/58690348304e8c9c91e76132e25a06db443883d1